tests/tracing/test_dnatracing_single_grain.py

```diff
@@ -96,21 +94,21 @@ def test_get_disordered_trace(
 
 # Currently two errors are not caught, need to improve this when refactoring, just in case.
 @pytest.mark.parametrize(
-    "min_skeleton_size, problem",
+    ("min_skeleton_size", "problem"),
     [
         (4, None),
         (4, 6),
     ],
 )
 def test_purge_obvious_crap_exceptions(dnatrace_linear: dnaTrace, min_skeleton_size: int, problem) -> None:
-    """Test exceptions to purge_obvious_crap"""
+    """Test exceptions to purge_obvious_crap."""
```

Comment:
We should probably change the name of this function at some point 😄

Reply:
Yes, I noticed that when refactoring `tracing/dnatracing.py` the test as it stands isn't actually testing anything (i.e. there is no `assert`ion being made).

Also the `pruge_obvious_crap` is no longer a method of the `dnaTracing` class. I think this is because with the restructuring to trace and skeletonise individual grains rather than the whole image this became redundant. Logged in #672.